doc/members/adding_member.rst

```diff
@@ -44,7 +44,7 @@ Activating a New Member
 
 A new member who gets registered in CCF is not yet able to participate in governance operations. To do so, the new member should first acknowledge that they are satisfied with the state of the service (for example, after auditing the current constitution and the nodes currently trusted).
 
-First, the new member should update and retrieve the latest state digest via the ``members/ack/update_state_digest`` command. In doing so, the new member confirms that they are satisfied with the current state of the service.
+First, the new member should update and retrieve the latest state digest via the ``gov/ack/update_state_digest`` command. In doing so, the new member confirms that they are satisfied with the current state of the service.
```

Comment:
I think this should be `/gov/ack...`, and all the `command`s should be `endpoint`.

Reply:
Done.